packages/portal-proto/src/features/annotations/filters.json

```diff
@@ -3,7 +3,7 @@
     "title": "Project",
     "field": "project",
     "full": "project.project_id",
-    "description": "No Description",
+    "description": null,
```

Comment:
fwiw, I don't think this is great. I would rather set description to optional but wasn't sure if there was a reason it's required.

Reply:
The FacetDefinition type is more describing what we get back from the _mapping endpoint. I'm not sure if they always have descriptions there? If not, it makes sense to have description as optional in the type. If we leave description as required, we can have the props for CreateFacetCard look for partial matches to the type like this:
```
index cde91864..f8a6443b 100644
--- a/packages/portal-proto/src/features/annotations/AnnotationFilterPanel.tsx
+++ b/packages/portal-proto/src/features/annotations/AnnotationFilterPanel.tsx
@@ -51,7 +51,7 @@ export const AnnotationFacetPanel = (): JSX.Element => {
         {FilterFacets.map((x) => {
           const facetName = x.title || fieldNameToTitle(x.full);
           return createFacetCard(
-            x as FacetDefinition,
+            x as Partial<FacetDefinition>,
             "Annotations",
             facetHooks,
             "annotation-browser",
diff --git a/packages/portal-proto/src/features/facets/CreateFacetCard.tsx b/packages/portal-proto/src/features/facets/CreateFacetCard.tsx
index ccde12cb..cb37e252 100644
--- a/packages/portal-proto/src/features/facets/CreateFacetCard.tsx
+++ b/packages/portal-proto/src/features/facets/CreateFacetCard.tsx
@@ -25,7 +25,7 @@ import { FacetRequiredHooks } from "@/features/facets/types";
  * @param width - override width of facet
  */
 export const createFacetCard = (
-  facet: FacetDefinition,
+  facet: Partial<FacetDefinition>,
   valueLabel: string,
   dataFunctions: FacetRequiredHooks,
   idPrefix: string,
amylehman@Amys-MBP gdc-frontend-framework % 
```

Reply:
I find it hard to believe that the _mapping endpoint will always return a description. Yet it's odd that `description` has been set to required and tests were passing. I'll investigate more with BE and decide which way to go based on findings. 

🙇‍♀️

Reply:
Ended up taking the `<Partial>` path. Seems like mappings will return an empty string for `description` whenever we don't actually have a description for the property.